templates/global/product-tab.php

```diff
@@ -20,17 +20,17 @@
                 <?php echo esc_html( $store_info['store_name'] ); ?>
             </span>
         </li>
-    <?php } ?>
+        <li class="seller-name">
+            <span>
+                <?php esc_html_e( 'Vendor:', 'dokan-lite' ); ?>
+            </span>
 
-    <li class="seller-name">
-        <span>
-            <?php esc_html_e( 'Vendor:', 'dokan-lite' ); ?>
-        </span>
+            <span class="details">
+                <?php printf( '<a href="%s">%s</a>', esc_url( dokan_get_store_url( $author->ID ) ), esc_attr( $store_info['store_name'] ) ); ?>
```

Comment:
kindly use numbered placeholder like `%1$s`, use the correct escape function for the store name.